### ISSUE TITLE
Spacing of chord symbols

### ISSUE THREAD

**Issue by Rabadman** [Jojo-Schmitz/MuseScore#489]
Hello Jojo,

I noticed the following with MU3.7 today. After selecting all chord symbols in a staff and increasing the Y offset in the inspector, all chord symbols are moved. With 3.6.2 or older MU3.7 version (7947196158) you could bring the chord symbols to the same height by increasing the Y-value.
Here is an example score with Y-offset 4.50sp. Open the file with MU3.6.2 and with the current MU3.7 and compare.

Windows 11 Home
Version 23H2
[Akkordsymbole.zip](https://github.com/Jojo-Schmitz/MuseScore/files/15448595/Akkordsymbole.zip)

**Comment by Jojo-Schmitz** [comment]
Build numbers (that 7947196158) don't help much, can you give the commit number from Help > About MuseScore?

**Comment by Rabadman** [comment]
I have checked my saved versions. I hope these are now the required version numbers.

Still works with version 3.7.0.8069549057 Evolution
Revision: e1f51e9

No longer works with version: 3.7.0.8182781437 Evolution
Revision: c459891

**Comment by Jojo-Schmitz** [comment]
Yep. that should help, thanks

**Comment by Jojo-Schmitz** [comment]
Looks to me as if you're 'miss-using' CHord symboils for Accodion Bass?
You don't use them for their sound for sure and have some underlined, others not and all just single note names and below staff.

**Comment by Jojo-Schmitz** [comment]
Also I can't seem to reproduce the issue from scratch

**Comment by Jojo-Schmitz** [comment]
I did, however, find the culprit commit:  e86e962, @worldwidewary?

**Comment by worldwideweary** [comment]
Yeah that is right. It's apparently a trade-off because of the way systems get collected and what happens during that process regarding harmony text. If you revert the commit, y-offsetting to make uniform a list selection of harmony text will work, but again at the expense of the chord symbols "dropping" on the 'next' system... 

Since the dropping is only temporary... And the y-offsetting is an actual functionality for having uniform y-spacing (also affects figured bass i think and RNA)... it is probably overall better to just revert the change.

**Comment by Rabadman** [comment]
Ich schreibe jetzt auf Deutsch. Wer möchte kann's eh übersetzen.
Besten Dank, dass du das so rasch korrigiert hast.
Korrekterweise hätte ich die Akkordsymbole in meiner Beispielpartitur im Inspecteur bei Akkordschichtung auf "nur Grundton" umstellen müssen. Bei den unterstrichenen Akkordsymbole handelt es sich laut meiner Frau um Terzbässe. Sie ist die Akkordeonistin. Bei Akkordeonnoten werden bei uns die Terzbässe mit Unterstrich dargestellt. Eine andere Darstellung der Akkordeonbässe in Musescore ist mir eigentlich nicht bekannt.

**Comment by Jojo-Schmitz** [comment]
Naja, kann man schon so machen, ist aber nicht ganz das das Akkordsymbole sind und kann daher auch schief gehen.
